R/type2.R

```diff
@@ -24,13 +24,14 @@
 #' but should be tested.
 #'
 #' Whenever you implemenet a `vec_type2.new_class()` generic/method,
-#' make sure to always provide `vec_type2.new_class.default()` (
-#' which should call [stop_incompatible_cast()]) and
+#' make sure to always provide `vec_type2.new_class.default()`
+#' (which should call [stop_incompatible_type()]) and
 #' `vec_type2.new_class.vctrs_unspecified()` (which should return `x`).
 #'
 #' See `vignette("s3-vector")` for full details.
 #' @keywords internal
-#' @param x,y Either vector types; i.e.
+#' @param x,y Vector types
+#' @param ... Unused
```

Comment:
This should be:

```
These dots are for future extensions and must be empty.
```

I have added an issue in ellipsis about documentation inheritance: https://github.com/r-lib/ellipsis/issues/22